### ISSUE TITLE
TLXst12 attributes

### ISSUE THREAD

**Issue by PaulTreleaven** [SMPTE/st2120-2#1]
In 11.2:
I believe the timeAddress range for Minutes and Seconds should be less than 60 ( or less than equal 59)

**Comment by PaulTreleaven** [comment]
In table 1, rationalType, I believe there is a typo: Both area numerator and a denominator are required  >> Both a numerator and a denominator are required

**Comment by PaulTreleaven** [comment]
I closed this issue in error and can't find a way to reopen it! The issue still needs attention.

**Comment by billredmann** [comment]
Agreed.  With respect to seconds and section A.2 Leap Second Corrections in ST 12-1, should we admit "less than or equal to 60" for the seconds field?  That seems permissible and would support facilities using that clause.  Further, it wouldn't interfere with facilities using the other recommendation (duplicative 59s for the last two seconds of the leap-second day).

**Comment by billredmann** [comment]
Thinking more on that, we should be consistent with the second inequality on each line...
< 24, <60, <61 (if that's what we choose) <30, so the subtly  of < vs <= isn't overlooked.
